PycurlClient/src/python/RestClient/ProxyPlugins/Socks5Proxy.py

```diff
@@ -1,4 +1,9 @@
-from urlparse import urlparse
+
+from future import standard_library
```

Comment:
This seems to be a leftover from the future compatibility as well, no?

Reply:
correct again